packages/infra/lib/surescripts/surescripts-stack.ts

```diff
@@ -288,12 +288,14 @@ export class SurescriptsNestedStack extends NestedStack {
     const convertPatientResponse = this.setupLambda("convertPatientResponse", {
```

Comment:
Infra bug fix for conversion Lambdas - couldn't use them since they didn't have access to the conversion bucket.